paasta_tools/utils.py

```diff
@@ -3090,6 +3122,23 @@ def get_running_mesos_docker_containers() -> List[Dict]:
     ]
 
 
+def create_service_boto_folder(service, keys):
+    service_folder = os.path.join(BOTO_SOURCE_DIR, service)
+    if not os.path.exists(service_folder):
+        os.mkdir(service_folder)
+    try:
+        for filename in os.listdir(BOTO_SOURCE_DIR):
+            if filename.split(".")[0] in keys:
+                os.link(
+                    os.path.join(BOTO_SOURCE_DIR, filename),
+                    os.path.join(BOTO_SOURCE_DIR, service, filename),
+                )
+    except OSError:
+        # Either BOTO_SOURCE_DIR is absent, or the keys inside already exist
+        # In either case, doing nothing is OK
+        pass
```

Comment:
just to be clear, if there are multiple instances of a service running on a box, they will all share the same directory, yeah, meaning they'll all have access to each others' keys, yeah?